### ISSUE TITLE
no longer automatically downloading private broadcasts

### ISSUE THREAD

**Issue by gizmodo999** [gitnew2018/My-OpenPeriscope#80]
Hey i noticed this issue earlier today that a private group i joined didnt start to automatically download like it has in the past... its also happened again this evening where i have to manually click download to get it downloading.. any ideas whats wrong?

**Comment by gitnew2018** [comment]
No idea, sorry.
If it happens again check the console for errors.

**Comment by clicksprocket** [comment]
don't know if it helps, but for me I had to 'select' the user - then it worked ( even though 'Selected user broadcasts' isn't checked in my automatic downloading options )

**Comment by owl46** [comment]
Are you using the last released version? I found out that after a while with the program open, the automatic downloads stopped working, as you've said.

I don't have this problem if i build the program step by step, instead of using the release version.

Hope it helps.

**Comment by balamut777** [comment]
I also confirm that the download has stopped working. The first version (which I had) just doesn't do anything, the latest release of My-OpenPeriscope v0. 2. 07 just creates an empty folder. When trying to download it manually it writes Failed to deserialize the V8 snapshot blob. This can mean that the snapshot blob file is corrupted or missing.
 Также подтверждаю, что скачивание перестало работать. Первая версия (которая у меня стояла) просто ничего не делает, последний релиз My-OpenPeriscope v0. 2. 07 просто создает пустую папку. При попытке скачать вручную  пишет Failed to deserialize the V8 snapshot blob. This can mean that the snapshot blob file is corrupted or missing..

**Comment by gizmodo999** [comment]
it started working again but what im noticing today is, a lot of the private broadcasts are downloaded numerous times... almost as if its downloading, stopping, then jjust starting again from the beginning. ive turned on debugging so now all i got to do is work out how to look for errors lol

**Comment by owl46** [comment]
> it started working again but what im noticing today is, a lot of the private broadcasts are downloaded numerous times... almost as if its downloading, stopping, then jjust starting again from the beginning. ive turned on debugging so now all i got to do is work out how to look for errors lol

Yes, i've also noticed this problem for the first time today. For one private broadcast i've had until 20 parallel downloads. A little scary! But maybe it is periscope acting funny today.

**Comment by ImControl** [comment]
Thought this issue was just with mine, downloaded 6+ of the same broadcast. Almost a couple gigs for a single broadcast.

**Comment by ghost** [comment]
Any updates on this? Seems to me its 50/50 whether or not it will autodownload a private broadcast.

**Comment by gitnew2018** [comment]
Not really, you could try owl46's suggestion and build the program step by step, instead of using the release version. 
I might have deleted to many "junk" files from nwjs.
My account is corrupted atm and not actively working on this, just reading github issues.

**Comment by balamut777** [comment]
I tried to build it myself, but the result is the same, it doesn't download, but only creates empty folders with the name. Maybe I'm doing something wrong. I follow the instructions on this page https://github.com/gitnew2018/My-OpenPeriscope/wiki/Build-Instructions the first option. Downloading NW.js (I tried nwjs-sdk-v0. 45. 5 and nwjs-sdk-v0. 46. 0-beta1), then download the archive https://github.com/gitnew2018/My-OpenPeriscope, getting files from the nwjs folder. I run the NW file, enter the SID in the field, everything starts and works except for writing.In Dowland manager on trance writes error: exit code: 3221225477. If you go to the Dowland tab and insert a link, it says Failed to open startup resource 'E:\Перископ\nwjs-sdk-v0.45.5-win-x64\v8_context_snapshot.bin'.
#
# Fatal error in , line 0
# Failed to deserialize the V8 snapshot blob. This can mean that the snapshot blob file is corrupted or missing._
#
#
#
#FailureMessage Object: 0000008695DFE210
1: 00007FFE7088110F node::MultiIsolatePlatform::CancelPendingDelayedTasks+5551
2: 00007FFE694E94A9 IsSandboxedProcess+2915733
3: 00007FFE6775187A v8::Isolate::Initialize+682
4: 00007FFE70856591 node::Buffer::New+499185
5: 00007FFE707D3C3F node::Start+559
6: 00007FFE66241A5F v8::internal::LargeObjectSpace::Available+1327
7: 00007FFE68C8CBDF v8::internal::TorqueGeneratedExportedMacrosAssembler::StoreSortStateTempArray+8942591
8: 00007FFE68CBD147 v8::internal::TorqueGeneratedExportedMacrosAssembler::StoreSortStateTempArray+9140583
9: 00007FFE68C8C8F4 v8::internal::TorqueGeneratedExportedMacrosAssembler::StoreSortStateTempArray+8941844
10: 00007FFE662411F2 ChromeMain+346
11: 00007FF7E5FF2931 Ordinal0+10545
12: 00007FF7E5FF1807 Ordinal0+6151
13: 00007FF7E6112392 GetHandleVerifier+750882
14: 00007FFEB5C837E4 BaseThreadInitThunk+20
15: 00007FFEB81BCB81 RtlUserThreadStart+33

**Comment by gitnew2018** [comment]
@balamut777 try nwjs v0.41.3 same as the release version. In theory it should work with newer nwjs (and older) but it's not guaranteed.

**Comment by balamut777** [comment]
I tried it, but the result is the same.

**Comment by gitnew2018** [comment]
Googling this "Failed to deserialize the V8 snapshot blob" tells me that this is something corrupted on your end. Maybe extracting nwjs is interrupted or it takes some corrupted files from temp.

**Comment by balamut777** [comment]
> Поиск в Google этого "не удалось десериализовать большой двоичный объект моментального снимка V8" говорит мне, что это что-то поврежденное на вашем конце. Возможно, извлечение nwjs прервано или он принимает некоторые поврежденные файлы из temp.

thanks for the tip. I started looking for a problem. I deleted the antivirus, but it didn't help. I downloaded the program to a flash drive and saved it to a flash drive - saving works. I specify saving to the hard disk, it also works, but if you transfer the program and save it to the hard disk, it does not save it. I haven't put any programs on recently.

**Comment by ghost** [comment]
> I tried to build it myself, but the result is the same, it doesn't download, but only creates empty folders with the name. Maybe I'm doing something wrong. I follow the instructions on this page https://github.com/gitnew2018/My-OpenPeriscope/wiki/Build-Instructions the first option. Downloading NW.js (I tried nwjs-sdk-v0. 45. 5 and nwjs-sdk-v0. 46. 0-beta1), then download the archive https://github.com/gitnew2018/My-OpenPeriscope, getting files from the nwjs folder. I run the NW file, enter the SID in the field, everything starts and works except for writing.In Dowland manager on trance writes error: exit code: 3221225477. If you go to the Dowland tab and insert a link, it says Failed to open startup resource 'E:\Перископ\nwjs-sdk-v0.45.5-win-x64\v8_context_snapshot.bin'.
> 
> # Fatal error in , line 0
> # Failed to deserialize the V8 snapshot blob. This can mean that the snapshot blob file is corrupted or missing._
> #FailureMessage Object: 0000008695DFE210
> 1: 00007FFE7088110F node::MultiIsolatePlatform::CancelPendingDelayedTasks+5551
> 2: 00007FFE694E94A9 IsSandboxedProcess+2915733
> 3: 00007FFE6775187A v8::Isolate::Initialize+682
> 4: 00007FFE70856591 node::Buffer::New+499185
> 5: 00007FFE707D3C3F node::Start+559
> 6: 00007FFE66241A5F v8::internal::LargeObjectSpace::Available+1327
> 7: 00007FFE68C8CBDF v8::internal::TorqueGeneratedExportedMacrosAssembler::StoreSortStateTempArray+8942591
> 8: 00007FFE68CBD147 v8::internal::TorqueGeneratedExportedMacrosAssembler::StoreSortStateTempArray+9140583
> 9: 00007FFE68C8C8F4 v8::internal::TorqueGeneratedExportedMacrosAssembler::StoreSortStateTempArray+8941844
> 10: 00007FFE662411F2 ChromeMain+346
> 11: 00007FF7E5FF2931 Ordinal0+10545
> 12: 00007FF7E5FF1807 Ordinal0+6151
> 13: 00007FF7E6112392 GetHandleVerifier+750882
> 14: 00007FFEB5C837E4 BaseThreadInitThunk+20
> 15: 00007FFEB81BCB81 RtlUserThreadStart+33

For me, the solution for the "Failed to deserialize the V8 snapshot blob" problem was the non latin characters in the folder name. It took me ages to figure it out. Try using a folder with no cyrillic characters.

**Comment by Kamouser** [comment]
Hi, I should start testing the OpenPeri ,I haven't used it in a while, i'm scared 
@gitnew2018 It works for you ?

**Comment by gitnew2018** [comment]
My periscope account got locked for some reason(i haven't used for ~half year) and i can't be bothered to make new one. You tell me if it works 😆.

**Comment by ghost** [comment]
> 
> 
> My periscope account got locked for some reason(i haven't used for ~half year) and i can't be bothered to make new one. You tell me if it works 😆.

I could easily give you some accounts. I'm used to making tons of backups

Edit: I also have the issue with it randomly stop autodownloading privates. It seems to only happen after it's been open for a while. Also, would be nice to have some sort of update if possible on group management via myopenperiscope. Would be cool to see since there is no where else to see a full list of all users in a group (since periscope shows only some and changes every refresh). Maybe a way to also disable any broadcasts from x country from ever showing? (LOL)

**Comment by gitnew2018** [comment]
@bootyislife nah 😅, i can make new account but my main problem is procrastination. When it feels like homework i just can't work on it.
As for more advanced group management you can try downloading kewalsk/My-OpenPeriscope, he did a ton of nice work on this.
You can hide by country but it's not permanent because i assumed people would set it and forget.

**Comment by ghost** [comment]
> 
> 
> @bootyislife nah 😅, i can make new account but my main problem is procrastination. When it feels like homework i just can't work on it.
> As for more advanced group management you can try downloading kewalsk/My-OpenPeriscope, he did a ton of nice work on this.
> You can hide by country but it's not permanent because i assumed people would set it and forget.

Ah I thought you couldn't sign in with that one
